build.gradle

```diff
@@ -18,7 +18,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.5.0'
+        classpath 'com.android.tools.build:gradle:2.1.2'
```

Comment:
@phmagic for now, leave the plugin version to `1.5.0`. From our last testing, any version >= 2.0.+ significantly increase the build time.